sklearn/feature_extraction/text.py

```diff
@@ -876,14 +876,19 @@ class CountVectorizer(_VectorizerMixin, BaseEstimator):
 
     ngram_range : tuple (min_n, max_n), default=(1, 1)
         The lower and upper boundary of the range of n-values for different
+
+        word n-grams or char n-grams to be extracted. All values of n such
+        that min_n <= n <= max_n will be used.
```

Comment:
You have twice "n-grams to be extracted. All values of n such that min_n <= n <= max_n will be used." in this paragraph.